src/bikesList.tsx

```diff
@@ -157,25 +168,25 @@ const  BikeListPage = () => {
               </ListSubheader>
               {list.map((bike,index)=>{
                 return (         
-                    <li key={bike.ID}>
+                    <li key={bike.id}>
                     <ListItem style={{backgroundColor: '#69696e',color:'white',display:'flex'}} onClick={()=>handleListItemClick(index)}>
                       <Box display="flex" flexDirection="row" p={1} m={1} alignSelf="center" style={{width:'90%'}}>
                         <Box p={0} width="5%"  >
-                          <ListItemText primary={bike.ID} ></ListItemText>
+                          <ListItemText primary={bike.id} ></ListItemText>
```

Comment:
Wywaliłbym całą kolumnę, ogólnie nie jest potrzebna, a przy id jako uuid to wgl krzak się robi